include/tins/pdu_cacher.h

```diff
@@ -53,7 +53,7 @@ namespace Tins {
  * etc.
  */
 template <typename T>
-class PDUCacher : public PDU {
+class TINS_API PDUCacher : public PDU {
```

Comment:
Is this needed for template classes? From the nature of templates, I would assume they shouldn't be imported by users of the library, as the full definition is always available.